src/arch.rs

```diff
@@ -57,6 +57,116 @@ registers!(Arm, {
     R13 = (13, "R13"),
     R14 = (14, "R14"),
     R15 = (15, "R15"),
+
+    WCGR0 = (104, "wCGR0"),
+    WCGR1 = (105, "wCGR1"),
+    WCGR2 = (106, "wCGR2"),
+    WCGR3 = (107, "wCGR3"),
+    WCGR4 = (108, "wCGR4"),
+    WCGR5 = (109, "wCGR5"),
+    WCGR6 = (110, "wCGR6"),
+    WCGR7 = (111, "wCGR7"),
+
+    ACC0 = (104, "ACC0"),
```

Comment:
These overlapping ranges give compilation warnings.

Probably what we need to do as add a `registers_alias!` macro for the aliases. So then `Arm::register_name` will only give the primary name, but we still define constants for the aliases.

Reply:
Oh that's true, I only did `cargo doc` and missed that. That sounds reasonable. I guess D0-D15 should be the primary names for 256-271 and wCGR0-wCGR7 for 104-111.

Reply:
I ended up reusing the `registers` macro so `name_to_register` will work if given the name of an aliasing register. I also updated the comment on `register_name`.